server/service/appconfig.go

```diff
@@ -175,6 +182,16 @@ func (svc *Service) ModifyAppConfig(ctx context.Context, p []byte) (*fleet.AppCo
 		}
 	}
 
+	// TODO: enforce unique group id?
+	var oldZendeskSettings []*fleet.ZendeskIntegration
+	if len(appConfig.Integrations.Jira) > 0 {
```

Comment:
```suggestion
	if len(appConfig.Integrations.Zendesk) > 0 {
```
? Are we missing a test to make this kind of typos pop?